### ISSUE TITLE
Public-api should query Prometheus directly

### ISSUE THREAD

**Issue by adleong** [runconduit/conduit#499]
Since the telemetry service is going away, the public-api service should be updated to query Prometheus directly.  The metrics available are described in #425 and #428.

The result should be that all data in the Conduit web dashboard and in the Conduit CLI stat command are provided by these new metrics.

**Comment by adleong** [comment]
Blocked on #428

**Comment by dadjeibaah** [comment]
Looking at the code, it looks like we can package most of functions in  `telemetry.go` and use the new package in the public-api server. The only concern is that we need to figure out (we may have already done so) what the prometheus queries would look like and what level of detail we want to show in the CLI and the web. 

Here are a couple of questions: 

- In the CLI, the stat command displays "request rate" is this rate associated with the inbound or outbound flow of traffic for a proxy?

- Do we still need to use the same protobuf definitions created for metrics queries e..g. Do we still need to use `telemetry.pb.go` and some of the metric definitions in `api.pb.go`? My guess is that we can get read of `telemetry.pb.go` and leave `api.pb.go` alone so we don't have to change the metric querying interface between the web and the public-api.

/cc @adleong thoughts?

**Comment by adleong** [comment]
1. inbound
2. that sounds right

**Comment by siggy** [comment]
Follow up from earlier discussion, we should consider moving to a model where the functionality to query Kubernetes and Prometheus is contained in a library, for both the web and cli to leverage.

**Comment by dadjeibaah** [comment]
I've got a branch that I think may no longer be needed since we are switching to the library model. Should I delete the branch or leave it as a reference?

**Comment by siggy** [comment]
@dadjeibaah I'd leave it as a reference until we at least ship the library.

**Comment by olix0r** [comment]
Ongoing in #671

**Comment by rmars** [comment]
This consists of moving 
- [ ] stat queries (#671) 
- [ ] ListPods(#694)
